src/HPRT/templates/base.html

```diff
@@ -68,19 +68,22 @@
 					<br>
 					<h1> <a href = "/"> HPRT </a> </h1>
 				</div>
+				{% if user.is_authenticated %}
 				<div class = "col-sm-10" align="right">
 					<br>
 					<a href="{% url 'patient_add_existing' %}" class="btn btn-light border" style="color:#007bff">+ Add Existing Patient</a>
 					<a href="{% url 'patient_new' %}" class="btn btn-light border" style="color:#007bff">+ New Patient</a>
-					
+
 					{% if user.user_type == 'A' or user.is_superuser %}
 					<a href="{% url 'signup' %}" class="btn btn-light border" style="color:#007bff">+ New User</a>
 					<a href="{% url 'site_new' %}" class="btn btn-light border" style="color:#007bff">+ New Site</a>
 					{% endif %}
 				</div>
+				{% endif %}
 
 			</div>
 			<div class="row" style="border-bottom:1px solid gray">
+				{% if user.is_authenticated %}
 				<div class = "col-sm-12" align="right">
```

Comment:
Also here, looks like there's some weird indentation going on in general with this block of code